test/ig/IGExporter.IG.test.ts

```diff
@@ -687,7 +730,7 @@ describe('IGExporter', () => {
       pkg.extensions.push(...pkgExtensions);
       pkg.logicals.push(...pkgLogicals);
       pkg.resources.push(...pkgResources);
-      exporter = new IGExporter(pkg, defs, path.resolve(fixtures, 'ig-data'), false);
+      exporter = new IGExporter(pkg, defs, path.resolve(fixtures, 'ig-data'));
```

Comment:
Nevermind. I see this was discussed below.

But for the record, I think I would like us to eventually clean this up.  But we don't need to do it in this PR.  Right now, I'd rather see this PR merged sooner than wait for this cosmetic change.

Reply:
If my searching was accurate, there are 70 files within various test fixture `ig-data` folders. I really want to update them, but it might also even just be easier to review changes if they're in a separate PR. Otherwise, I do think it would be doable here - it doesn't seem difficult.

Reply:
I want to add my vote to doing it another time.

Reply:
Yep.  Let's do it later.

Reply:
Perfect 🙏 

I'll plan to make a PR post-2.0.